sormas-app/app/src/main/res/layout/fragment_immunization_new_layout.xml

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Please have another look at the development specifications and remove the fields that are only supposed to be part of the edit form

Reply:
can you please confirm that:
the number of doses be in the new layout without the heading
recovery related fields should should not be in the new layout

Reply:
@BarnaBartha Yes, exactly!